### ISSUE TITLE
Hard-coded wlan0 interface in device configure modal

### ISSUE THREAD

**Issue by sanpii** [Digilent/waveforms-live#31]
Hello,

When I open the device configure modal, to display wifi informations, waveforms send a `nicGetStatus` request for `wlan0` adpater regardless to `nicList` content.

**Comment by SudharsanSukumar** [comment]
Hi,

I've looked at the issues you opened. Are you trying to add WFL support for a new device?

**Comment by sanpii** [comment]
> Are you trying to add WFL support for a new device?

Yes, I opened those issues mainly for reference. It’s not critic.
